pkg/errors/errors.go

```diff
@@ -26,6 +26,10 @@ var ErrInvalidWorkflowType = errors.New("invalid workflow type, must be either '
 var ErrInvalidProcessType = errors.New("invalid process type, must be either 'trigger', 'task' or 'exit'")
 var ErrInvalidProcessObjectStoreScope = errors.New("invalid process object store scope, must be either 'product' or 'workflow'")
 var ErrInvalidNetworkingProtocol = errors.New("invalid networking protocol, must be either 'UDP' or 'TCP'")
+var ErrInvalidProcessCPU = errors.New("invalid process CPU, must be of form '1', '0.5' or '100m'")
+var ErrInvalidProcessCPURelation = errors.New("invalid process CPU relation, 'limit' cannot be lower than 'request'")
+var ErrInvalidProcessMemory = errors.New("invalid process memory, must be of form '350M' or '1Gi'")
```

Comment:
Same as CPU, we should reference that this is a resource limit value somewhere in the message.

```suggestion
var ErrInvalidProcessMemory = errors.New("invalid process memory limit, must be of form '350M' or '1Gi'")
```

Reply:
Ditto, refers to both values